INSTRUCTION.md

```diff
@@ -0,0 +1,22 @@
+## How to validate App is running
+``
+     http://localhost:30007/api
+``
+
+## How to validate configMap data is mounted as files in the right order
+## Secret data is mounted as a file
+
+``
+     
+     kubectl get pods
+    kubectl exec {pod} -it -- sh 
```

Comment:
Consider providing guidance on how to dynamically retrieve the pod name using a command like `kubectl get pods -l app=todoapp -o jsonpath='{.items[0].metadata.name}'` to make it easier for users to execute commands without manually finding the pod name.